### DISCUSSION TITLE
v2.1.6

### DISCUSSION THREAD

**Discussion by clementwzk** [clementwzk/OpenCalc#231] [Announcements]
**Fixed:**
- tan() does not work properly (#229)
- Deleting a thousand separator causes the cursor to jump (#218)
- Placement of parentheses inside existing calculations (#213)
- Graphics problem in landscape mode for tablets over 10 inches (#160)

<hr /><em>This discussion was created from the release <a href='https://github.com/Darkempire78/OpenCalc/releases/tag/v2.1.6'>v2.1.6</a>.</em>

**Comment by mmtxrc** [comment]
you see how google calculator does that thing where the buttons turn into rounded squares when pressed/long-pressed? is that possible to implement in opencalc?

**Reply by clementwzk** [reply]
open an issue for that please
